docs/QUICKSTART.md

```diff
@@ -60,81 +60,75 @@ Next we install the TestSys namespace, controller and CRD schemas into the TestS
-  --ami "${AMI_ID}" \
-  --ec2-provider-image "ec2-resource-agent:eks"
+cd bottlerocket/samples
+export CLUSTER_NAME=${EKS_CLUSTER_NAME}
+export ARCH="x86_64"
+export VARIANT="aws-k8s-${K8S_VER}"
+export CLUSTER_TYPE="kind"
+export SONOBUOY_TEST_AGENT_IMAGE_URI="sonobuoy-test-agent:eks"
+export EKS_RESOURCE_AGENT_IMAGE_URI="eks-resource-agent:eks"
+export EC2_RESOURCE_AGENT_IMAGE_URI="ec2-resource-agent:eks"
+cargo make run sonobuoy-test
 ```
 
 ## Test Results
 
-Executing the `testsys run aws-k8s` command will kick off the setup and execution of the Sonobuoy tests on an EKS cluster using Bottlerocket as the host OS for the cluster nodes.
+Executing the `cargo make run sonobuoy-test` command will kick off the setup and execution of the Sonobuoy tests on an EKS cluster using Bottlerocket as the host OS for the cluster nodes.
 
-To get the status and results of test execution, run the `testsys status` command for a high level summary:
+To get the status and results of test execution, run the `cli status` command for a high level summary:
 
 ```shell
- NAME           TYPE   STATE    PASSED   SKIPPED   FAILED
- testsys-demo   Test   passed   1        5772      0
+ NAME                         TYPE       STATE       PASSED   SKIPPED   FAILED
+ external-cluster             Resource   completed                                                                                                                                                                                      
+ external-cluster-instances   Resource   completed
+ external-cluster-test        Test       passed      1        5772      0
 ```
 
-**Note:** run `testsys status --help` to learn about more options.
+**Note:** run `cli status --help` to learn about more options.
 Notably the `-c` and `-r` arguments for getting controller and resource status.
 
 When the test run has completed you may get the full logs of the test results.
 The content of the resulting tar file will vary depending on the tests being run.
 
 ```shell
-testsys results --destination testresults.tar --test-name testsys-demo
+cli results --destination testresults.tar --test-name testsys-demo
 tar -xvf testresults.tar
 ```
 
 ## Cleanup
 
-If you used the `--cluster-destruction-policy never` as given above, there will be an EKS cluster running at the end of execution.
+Most of the cleanup can be done by using this command:
+
+```shell
+cli delete
+```
+
+This will delete all tests and resources where `destructionPolicy` is `onDeletion`.
+
+If you used `destructionPolicy: never`, there will be an EKS cluster running at the end of execution.
 This can be very convenient to keep around for subsequent test runs.
 
 It does however consume resources and incur usage charges.
-If you are done running tests and would like to clean up, run the following to delete the EKS cluster.
+If you are done running tests and would like to clean up the EKS cluster, run the following to delete it.
```

Comment:
Don't the samples automatically clean up resources?

Reply:
Yeah, this follows line 127 where `destructionPolicy: never` is specified in case the user manually changed it themselves. I can remove that line but I thought it would be a good idea to leave it in in case the user wanted to keep a cluster around and couldn't figure out how to delete it afterwards.